src/envoy/mixer/http_control.h

```diff
@@ -37,7 +37,8 @@ typedef std::shared_ptr<HttpRequestData> HttpRequestDataPtr;
 class HttpControl final : public Logger::Loggable<Logger::Id::http> {
  public:
   // The constructor.
```

Comment:
I created a new PR after I re-base the repo

Please review the new one https://github.com/istio/proxy/pull/93